vignettes/articles/multi-scale-resampling.Rmd

```diff
@@ -0,0 +1,233 @@
+---
+title: "Multi-scale model assessment with spatialsample"
+---
+
+```{r include = FALSE}
+knitr::opts_chunk$set(
+  cache = TRUE,
+  cache.lazy = FALSE,
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+
+Modeling spatially structured data is complicated. In addition to the usual difficulty of statistical modeling, models of spatially structured data may have spatial structure in their errors, with different regions being more or less well-described by a given model. This also means that accuracy metrics for these models might change depending on what spatial scale is being assessed. Only investigating model accuracy at larger aggregation scales, such as when accuracy is only assessed for the entire study area as a whole, might "smooth out" these local differences and present an inaccurate picture of model performance.
+
+For this reason, a number of researchers (most notably, [Riemann et al. (2010)](https://www.nrs.fs.fed.us/pubs/jrnl/2010/nrs_2010_riemann_001.pdf)[^1]) have suggested assessing models at multiple scales of spatial aggregation to ensure cross-scale differences in model accuracy are identified and reported. This post walks through how to do that using the new [spatialsample](https://spatialsample.tidymodels.org/) package.
+
+[^1]: Riemann, R., Wilston, B. T., Lister, A., and Parks, S. 2010. An effective assessment protocol for continuous geospatial datasets of forest characteristics using USFS Forest Inventory and Analysis (FIA) data. Remote Sensing of Environment, 114, pp. 2337-2353. doi: 10.1016/j.rse.2010.05.010.
+
+Because Riemann et al. were working with data from the US Forest Inventory and Analysis (FIA) program, we're going to do the same. However, because our main goal is to show how spatialsample can support this type of analysis, we won't spend a ton of time worrying about any of the quirks of FIA data[^2] or on feature engineering. Instead, we're going to use a simple linear model to see if we can predict how much aboveground biomass ("AGB"; all the non-root woody bits) there is in a forest based on how many trees there are. We'll use all the FIA field data from New York State, USA. 
+
+[^2]: Among them that only forested areas are measured, where "forested" means "principally used as forest" which excludes parks but includes recently clear-cut lands, and that plot locations are considered personally identifying information under the farm bill of 1985, and so as to not identify anyone the coordinates in public data are "fuzzed" by a few miles and approximately 20% of plot coordinates are swapped with other plots in the data set. Which is to say, consult your local forester or ecologist if you want to use FIA data to answer real questions in your own work.
+
+Because we're mostly interested in assessing our models, I'm going to mostly ignore how exactly I downloaded and wrangled the FIA data for this post. If you're curious, I've hidden the code below:
```

Comment:
```suggestion
Because we're mostly interested in assessing our models, let's not focus on how exactly to download and wrange the FIA data. If you're curious, the code is in a hidden chunk here:
```